tests/run.sh

```diff
@@ -179,12 +169,11 @@ if [[ -n $SPECIFIC_INTEGRATION_TEST ]]; then
 fi
 
 python2 -m pytest \
-    $XDIST_ARGS \
-    $MAX_FAIL_ARG \
-    -s \
-    --verbose \
-    --junitxml=results.xml \
-    $HTML_REPORT \
-    $pass_args \
-    $SPECIFIC_INTEGRATION_TEST_FLAG "$SPECIFIC_INTEGRATION_TEST" \
-    $DEFAULT_TESTS
+        $XDIST_ARGS \
+        $MAX_FAIL_ARG \
```

Comment:
More room for improvement: these two are complementary, we will have either one or the other. I suggest merging them. The ugly but backwards compatible way would be to just set `XDIST_ARGS="--maxfail=1"` when pytest is not installed. The nicer way would be with a new variable, but then you have to fix mender-qa as well.

Reply:
Alright! I like it. I think I'll create a seperate PR for it though, as then I am able to separate this PR from running it alongside `mender-qa` with a separate PR :)

Reply:
Follow-up task: https://tracker.mender.io/browse/QA-121